### PR TITLE
Add new controls closes #50

### LINKED ISSUE #50: Add new controls
**Is your feature request related to a problem? Please describe.**

Add below below controls:
- container_registry_zone_redundant
- container_group_secure_environment_variable

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by khushboo9024**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

**Review by bigdatasourav — APPROVED** [review]

File: query/containerregistry.sp [hunk 1]
```diff
@@ -267,3 +267,40 @@ query "container_registry_trust_policy_enabled" {
...
+      ${local.tag_dimensions_sql}
+      ${local.common_dimensions_sql}
+    from
+      terraform_resource as r
+      left join geo_replication_zone_redundant as g on g.name = r.name
+    where
+      type = 'azurerm_container_registry';
+  EOQ
+}
+
```
Comment (madhushreeray30) [inline query/containerregistry.sp]: @khushboo9024 I am getting an error while running this query
```
madhushreeray ~/turbot/steampipe-mod-terraform-azure-compliance [issue-50] $ steampipe check control.container_registry_zone_redundant_enabled                     

+ Container registries should be zone redundant ..................................................................................................................................... 1 / 1 [==========]
  
  ERROR: cannot extract elements from an object (SQLSTATE 22023)
  ```

File: query/containerinstance.sp [hunk 2]
```diff
@@ -18,3 +18,35 @@ query "container_instance_container_group_in_virtual_network" {
...
+        and c -> 'environment_variables' is not null
+    )
+    select
+      address as resource,
+      case
+        when e.name is not null then 'alarm'
+        else 'ok'
+      end status,
+      split_part(address, '.', 2) || case
+        when e.name is not null then ' uses environment variables.'
```
Comment (madhushreeray30) [inline query/containerinstance.sp]:
```suggestion
        when e.name is not null then ' uses environment variables'
```

File: query/containerinstance.sp [hunk 3]
```diff
@@ -18,3 +18,35 @@ query "container_instance_container_group_in_virtual_network" {
...
+        else ' does not use environment variables.'
```
Comment (madhushreeray30) [inline query/containerinstance.sp]:
```suggestion
        else ' does not use environment variables'
```

**Review by madhushreeray30 — COMMENTED** [review]
@khushboo9024 Please take a look at the comments, thanks!

**Review by madhushreeray30 — APPROVED** [review]

**Pull request merged**